frontend/src/metabase/query_builder/components/LimitInput.tsx

```diff
@@ -21,7 +21,7 @@ const LimitInput = ({
   ...props
 }: LimitInputProps) => (
   <input
-    className={cx(CS.input, className, {
+    className={cx(CS.input, className, CS.bgWhite, {
```

Comment:
Fix the row limit being broken in dark mode. Previously it shows as #fff in dark mode.

![CleanShot 2568-01-28 at 22 32 49@2x](https://github.com/user-attachments/assets/81edcd81-4870-4dee-82c7-83ce5f1677b4)

Reply:
I think I think this somewhere, why does `bg-white` work here? Isn't it supposed to be white?